scalableminds/webknossos
# in script update route, dont return temporary id

### Steps to test:
- update a script, see in network tab that returned json contains correct _id

------
